tests/python/contrib/test_vitis_ai/test_vitis_ai_runtime.py

```diff
@@ -101,4 +103,7 @@ def test_extern_vai_resnet18():
     check_result(mod, {"data": i_data},
                  (1, 1000), ref_res.asnumpy(), tol=1e-5, params=params)
 if __name__ == "__main__":
-    test_extern_vai_resnet18()
+    if sys.platform == "win32":
+        print("Skip test on Windows for now")
+    else:
+        test_extern_vai_resnet18()
```

Comment:
```suggestion
    if sys.platform == "win32":
        print("Skip test on Windows for now")
        return
    test_extern_vai_resnet18()
```

Reply:
This actually gives a linter error, that's why we switched to the if/else